feeds/product.php

```diff
@@ -260,23 +261,36 @@ function arrayMergeByIdProduct($array1 = [], $array2 = [])
 $header = array_merge($header, $extraHeader);
 // To avoid indexation failures
 $header = array_unique($header);
-
-// PRODUCTS
-$rows = DfTools::getAvailableProductsForLanguage($lang->id, $shop->id, $limit, $offset);
-
-$rows = arrayMergeByIdProduct($rows, $extraRows);
-
-// In case there is no need to display prices, avoid calculating the mins by variant
-$minPriceVariantByProductId = ($shouldShowProductVariations && $shouldDisplayPrices) ? DfTools::getMinVariantPrices($rows, $shouldPricesUseTaxes, $currencies, $lang->id, $shop->id) : [];
 $additionalHeaders = array_merge($additionalAttributesHeaders, $extraHeader);
 
 $csv = fopen('php://output', 'w');
 if (!$limit || (false !== $offset && 0 === (int) $offset)) {
     fputcsv($csv, $header, DfTools::TXT_SEPARATOR);
 }
 
-foreach ($rows as $row) {
-    $product = $dfProductBuild->buildProduct($row, $minPriceVariantByProductId, $additionalAttributesHeaders, $additionalHeaders);
+$products = DfTools::getAvailableProducts($lang->id, $shouldShowProductVariations, $limit, $offset);
+$empty_line = array_fill_keys($header, null);
```

Comment:
```suggestion
$emptyLine = array_fill_keys($header, null);
```

Where are you using this variable? :thinking: